packages/react-core/src/components/Title/examples/Title.md

```diff
@@ -7,55 +7,9 @@ propComponents: ['Title']
 
 ## Examples
 ### Sizes
-```js
-import React from 'react';
-import { Title, TitleSizes } from '@patternfly/react-core';
-
-<React.Fragment>
-  <Title headingLevel="h1" size={TitleSizes['4xl']}>
-    4xl Title
-  </Title>
-  <Title headingLevel="h2" size="3xl">
-    3xl Title
-  </Title>
-  <Title headingLevel="h3" size={TitleSizes['2xl']}>
-    2xl Title
-  </Title>
-  <Title headingLevel="h4" size="xl">
-    xl Title
-  </Title>
-  <Title headingLevel="h5" size={TitleSizes.lg}>
-    lg Title
-  </Title>
-  <Title headingLevel="h6" size="md">
-    md Title
-  </Title>
-</React.Fragment>
+```ts file="./TitleSizes.tsx"
```

Comment:
Related to the comments above, this file name should match the export within the file. So this should be updated to `TitleCustomSizes.tsx`, as well as the file itself.